tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/util/ProjectImportUtil.java

```diff
@@ -42,6 +43,10 @@
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.io.filefilter.DirectoryFileFilter;
 import org.apache.commons.io.filefilter.WildcardFileFilter;
+import org.eclipse.buildship.core.projectimport.ProjectImportConfiguration;
```

Comment:
but gradle tools is the core function in IDE 3.0 M1 , if users do not choose gradle tools , that means they can't create modules for liferay 7.